apiserver/facades/client/modelmanager/register.go

```diff
@@ -86,6 +90,16 @@ func newFacadeV10(stdCtx context.Context, ctx facade.MultiModelContext) (*ModelM
 	backend := common.NewUserAwareModelManagerBackend(configSchemaSource, model, pool, apiUser)
 
 	secretBackendService := domainServices.SecretBackend()
+
+	bootstrapParamsData, err := os.ReadFile(path.Join(ctx.DataDir(), cloudconfig.FileNameBootstrapParams))
```

Comment:
Why are we doing this here? It's a point in time which may change. As well, the cloud will not be correct if the user is setting up a multi-cloud controller.

Another consideration is that very little of that data is actually used.

Reply:
Outdated, metadata are now copied from controller model db